src/pages/AppList.svelte

```diff
@@ -16,12 +16,20 @@
 
   onMount(async () => {
     apps = await fetchJSON(URL);
-    apps.sort((a, b) => (a.name > b.name ? 1 : -1));
+    apps.sort((a, b) =>
+      a.canonical_app_name.toLowerCase() > b.canonical_app_name.toLowerCase()
```

Comment:
This is somewhat arbitrary, but seems to give better results than sorting on the capitalized versions of things. Filed #456 to fix this in a more principled way.